framework/src/controller/schema/application.js

```diff
@@ -13,6 +13,211 @@ module.exports = {
 	constants: {
 		id: '#/app/constants',
```

Comment:
This is huge and major application element, so create it as `schema/constants.js` file.